datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.tsx

```diff
@@ -402,6 +554,58 @@ export const TABLE_DENY: RecipeField = {
         setListValuesOnRecipe(recipe, values, tableDenyFieldPath),
 };
 
+const chartAllowFieldPath = 'source.config.chart_pattern.allow';
+export const CHART_ALLOW: RecipeField = {
+    name: 'chart_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    type: FieldType.LIST,
+    fieldPath: 'source.config.chart_pattern.allow',
```

Comment:
something I've noticed all over this file is the field path is duplicated. So we can use the constant defined above the field object as the `fieldPath` value.

ie. above can become:
```
const chartAllowFieldPath = 'source.config.chart_pattern.allow';
export const CHART_ALLOW: RecipeField = {
    name: 'chart_pattern.allow',
    label: 'Allow Patterns',
    tooltip: 'Use regex here.',
    type: FieldType.LIST,
    fieldPath: chartAllowFieldPath,
``` 

Would you mind making this update elsewhere as well?

Reply:
Checked & Fixed